org.gnome.Music.json

```diff
@@ -88,34 +81,35 @@
             ],
             "sources": [{
                 "type": "archive",
-                "url": "https://download.gnome.org/sources/grilo/0.3/grilo-0.3.4.tar.xz",
-                "sha256": "7c6964053b42574c2f14715d2392a02ea5cbace955eb73e067c77aa3e43b066e"
+                "url": "https://download.gnome.org/sources/grilo/0.3/grilo-0.3.6.tar.xz",
+                "sha256": "1e65ca82dd58020451417fde79310d4b940adc3f63ab59997419c52ed3bc9c91"
             }],
             "cleanup": ["/include", "/bin"]
         },
         {
             "name": "grilo-plugins",
+            "buildsystem": "meson",
             "config-opts": [
-                "--enable-tracker",
-                "--enable-dleyna",
-                "--enable-goa",
-                "--enable-filesystem=no",
-                "--enable-optical-media=no",
-                "--enable-youtube=no",
-                "--enable-bookmarks=no",
-                "--enable-lua-factory=no",
-                "--enable-metadata-store=no",
-                "--enable-vimeo=no",
-                "--enable-localmetadata=no",
-                "--enable-thetvdb=no",
-                "--enable-tmdb=no",
-                "--enable-freebox=no",
-                "--enable-opensubtitles=no"
+                "-Denable-tracker=yes",
+                "-Denable-dleyna=yes",
+                "-Denable-goa=yes",
+                "-Denable-filesystem=no",
+                "-Denable-optical-media=no",
+                "-Denable-youtube=no",
+                "-Denable-bookmarks=no",
+                "-Denable-lua-factory=no",
```

Comment:
Upstream this got enabled past 3.30.1 (https://gitlab.gnome.org/GNOME/gnome-music/commits/gnome-3-30) to enable coverart lookup (via The AudioDB/lastfm). Although this may have some issues still, it is recommended to include it.

Reply:
Okay, so I should enable lua-factory? :)

Reply:
That's the small hickup, the subproject (the flathub lua json) is not in the 3.30.1 package. It got added afterwards. You probably know best how to add it to the flathub json.

Also, post 3.30.1 a few fixes were added to resolve some of the issues with the flatpak version and cover lookup. So maybe this has to wait to 3.30.2, but that is still a fortnite away.

Reply:
We can definitely push that as an update of the same version.  Especially that there are other dependencies that are not needed anymore.

Reply:
Or I could make a 3.30.1.1 just for the flatpak changes. But that will have to wait until the weekend as well, got no time before that.

Reply:
Will update the package here, for now, once the 3.30.1.1 or 3.30.2 is released will update here with the correct dependencies already in place.